blockchain/state/statedb.go

```diff
@@ -397,12 +398,12 @@ func (s *StateDB) IsValidCodeFormat(addr common.Address) bool {
 	return false
 }
 
-// GetVmVersion return false when getStateObject(addr) or GetProgramAccount(stateObject.account) is failed.
+// GetVmVersion return false when getStateObject(addr) or GetProgramAccount(stateObject.account) fails and the account is EOA without code.
```

Comment:
```suggestion
// GetVmVersion return false when getStateObject(addr) or GetProgramAccount(stateObject.account) fails or the account is EOA without code.
```

Reply:
> "(GetProgramAccount(stateObject.account) fails) && (the account is EOA without code)"

Since the conditions are as above, isn't it "and"?

Reply:
This is how I understood:

GetVmVersion return _false_ when
- getStateObject(addr) fails
- or GetProgramAccount(stateObject.account) fails   (i.e. pa == nil)
- or the account is EOA without code.  (i.e. bytes.Equal)

Having said that, I think it's better to state the inverse of the predicate:
GetVmVersion returns _true_ when the address is an SCA or an EOA with code.

Reply:
> Having said that, I think it's better to state the inverse of the predicate:
GetVmVersion returns true when the address is an SCA or an EOA with code.

That's a good idea. I've fixed in https://github.com/kaiachain/kaia/pull/166/commits/c0d6d188868731b29a20c1bd96d2bb02432d84b7.